stories/s1grdiw.eocloud.stories.js

```diff
@@ -214,6 +214,49 @@ export const findTilesEPSG4326 = () => {
   return wrapperEl;
 };
 
+export const findFlyovers = () => {
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = "<h2>findFlyovers</h2>";
+
+  const img = document.createElement('img');
+  img.width = '512';
+  img.height = '512';
+  wrapperEl.insertAdjacentElement("beforeend", img);
+
+  const flyoversContainerEl = document.createElement('pre');
+  wrapperEl.insertAdjacentElement("beforeend", flyoversContainerEl);
+
+  const perform = async () => {
+    const layer = (await LayersFactory.makeLayers(`${DATASET_EOCLOUD_S1GRD.shServiceHostname}v1/wms/${instanceId}`, (lId, datasetId) => (layerId === lId)))[0];
```

Comment:
That doesn't work. Probably wrong layer name, other examples in this story also fails with 500 because of nonexistent layer `S1`

Reply:
The `.env` example in internal documentation had the wrong value, which caused this example not to work (I have already fixed it). I think if you update `.env` it should work.